test/framework/src/main/java/org/elasticsearch/test/AbstractQueryTestCase.java

```diff
@@ -858,21 +854,17 @@ protected static String getRandomRewriteMethod() {
     }
 
     private static String[] getRandomTypes() {
-        String[] types;
```

Comment:
I just saw this is also tagged with 6.3, I think this part of the change shouldn't be backported because in 6.x we should still test for multiple types (support for 5.x indices)